packages/blueprint-component-mapper/src/dual-list-select/dual-list-select.d.ts

```diff
@@ -26,6 +26,7 @@ interface InternalDualListSelectProps {
   filterOptionsText?: ReactNode;
   leftValues?: DualListSelectValue[];
   rightValues?: DualListSelectValue[];
+  isFilterCaseInSensitive?: boolean;
```

Comment:
I would suggest to not use prop for it and set it as a default behavior. 

@Hyperkid123 what do you think?

Reply:
I'd go for default as well.

Reply:
Raising new PR: https://github.com/data-driven-forms/react-forms/pull/1473, closing this one.